frontend/public/components/RBAC/role.jsx

```diff
@@ -145,7 +145,7 @@ export const BindingsForRolePage = (props) => {
     staticFilters={[{'role-binding-roleRef': name}]}
     resources={[
       {kind: 'RoleBinding', namespaced: true},
-      {kind: 'ClusterRoleBinding', namespaced: false},
+      {kind: 'ClusterRoleBinding', namespaced: false, optional: true},
```

Comment:
Maybe we should have some conditional logic here to choose between cluster level or namespaced role bindings depending on which type of role the user is viewing?

Reply:
Yeah, I was thinking that. A ClusterRoleBinding can't reference a namespaced Role, so we don't need to list ClusterRoleBindings in that case.

https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/rbac/validation/validation.go#L179